RPIPs/RPIP-49.md

```diff
@@ -107,8 +109,9 @@ The tokenomics rework package will likely be split between two protocol upgrades
 * [RPIP-59: Deposit Mechanics](RPIP-59.md)
   * Standard and express queues.
 * [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
-  * All but heuristic adjustments.
-* RPL Value Capture - [RPL Burn](RPIP-45.md) / [RPL Buy & LP](RPIP-50.md) / Increased share to voting Node Operators.
+  * Splits between rETH, NOs, and vote-eligible staked RPL
+  * RPL issuance rewards no longer have a minimum stake required
+* RPL Value Capture - Increased share to voting Node Operators.
```

Comment:
Same comment... generic description, so probably good enough (instead of trying to be technically accurate and more specific)?

Reply:
I used your earlier suggestion "vote-eligible RPL staked in megapools"